unit_test/main_tests/conftest.py

```diff
@@ -4,12 +4,7 @@
 
 import pytest
 
-from cibuildwheel import (
-    linux,
-    macos,
-    util,
-    windows,
-)
+from cibuildwheel import linux, macos, util, windows
```

Comment:
This is irritating, it didn't respect Black's magic comma.

Reply:
I guess it doesn't. Sad. For things like typing I add the comma because I know they are likely to change. (Not true of this case, of course, but might be useful elsewhere, especially with typing).

Reply:
> This is irritating, it didn't respect Black's magic comma.

What did you mean? Black put `,` only at the end of the multiline import. When the import is a single line then black does not put a comma on the end.

Reply:
True, that would be nice, if we could teach it. And once again (I'm surprised) I would agree with Black that the comma there is a pretty good indicator to say that this is a changing list.